tests/constants.js

```diff
@@ -18,13 +19,20 @@ module.exports = {
   MAIN_DDOC_NAME: 'medic',
 
   // tests create a document with this id to be referenced by org.couchdb.user contact_id
-  USER_CONTACT_ID: 'e2e_contact_test_id',
+  USER_CONTACT_ID,
 
   DEFAULT_USER_CONTACT_DOC: {
-    _id: 'e2e_contact_test_id',
+    _id: USER_CONTACT_ID,
     type: 'person',
     reported_date: 1541679811408,
   },
+  DEFAULT_USER_ADMIN_TRAINING_DOC: {
```

Comment:
So, looking at the way we check that the training forms have been completed, we search by id prefix and we _don't care_ what type of document this is. 
So this can have any _type_, it doesn't have to be a `data_record`. 

https://github.com/medic/cht-core/blob/e33cd8738be9bbe1e3c34566fe07d7bf9fa9da6f/webapp/src/ts/services/training-cards.service.ts#L99-L109

It's a big cheat! but I think it could work and wouldn't require all the deletions and report exclusion magic.

Reply:
This approach is cursed.  And I love it!  :sweat_smile: Trying it now. :+1: